src/normal.c

```diff
@@ -7568,12 +7568,14 @@ nv_put_opt(cmdarg_T *cap, int fix_indent)
 		reg1 = get_register(regname, TRUE);
 	    }
 
+#if defined(FEAT_CLIPBOARD) || defined(PROTO)
```

Comment:
`defined(PROTO)` is not needed inside a function.
It is needed for generating function prototypes using the cproto program.

Reply:
OK.  Fixed.